helm_deploy/values-preprod.yaml

```diff
@@ -1,72 +1,60 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 4
 
-image:
-  repository: quay.io/hmpps/manage-key-workers
-  tag: latest
-  pullPolicy: IfNotPresent
-  port: 3000
+generic-service:
+  replicaCount: 4
+  ingress:
+    host: preprod.manage-key-workers.service.justice.gov.uk
+  env:
+    API_ENDPOINT_URL: https://api-preprod.prison.service.justice.gov.uk
+    OAUTH_ENDPOINT_URL: https://sign-in-preprod.hmpps.service.justice.gov.uk/auth
+    KEYWORKER_API_URL: https://keyworker-api-preprod.prison.service.justice.gov.uk
+    NN_ENDPOINT_URL: https://digital-preprod.prison.service.justice.gov.uk/
+    PRISON_STAFF_HUB_UI_URL: https://digital-preprod.prison.service.justice.gov.uk/
+    HMPPS_COOKIE_NAME: hmpps-session-preprod
+    TOKENVERIFICATION_API_URL: "https://token-verification-api-preprod.prison.service.justice.gov.uk"
+    SUPPORT_URL: https://support-preprod.hmpps.service.justice.gov.uk/
+    COMPLEXITY_OF_NEED_URI: https://complexity-of-need-preprod.hmpps.service.justice.gov.uk
+    PRISONS_WITH_OFFENDERS_THAT_HAVE_COMPLEX_NEEDS: "AGI,BZI,DHI,DWI,ESI,EWI,FHI,LNI,NHI,PFI,SDI,STI"
+  allowlist:
+    office: "217.33.148.210/32"
```

Comment:
I can't see why this list is different to all the others..

BUT it's missing the `global-protect` one, which is already in prod (it was added to prod a while ago)